package.json

```diff
@@ -103,5 +103,9 @@
     "fail": [
       "@semantic-release/github"
     ]
+  },
+  "devDependencies": {
+    "@testing-library/react-hooks": "^1.1.0",
+    "react-test-renderer": "^16.8.6"
```

Comment:
We are not using dev dependencies because our build tool only includes the used code in production.

Reply:
Fixed.